### PR TITLE
Configure test cluster to use loki

### LINKED ISSUE #324: Setup Logging on the New Observability Cluster 2/2
- subtask of https://github.com/nerc-project/operations/issues/177
- Zenhub Epic [```Setup a new Cluster to handle Metrics, Logging and Observability```](https://app.zenhub.com/workspaces/moc-a--rh-sprint-workspace-62a210f69d42f600151deae0/epics/Z2lkOi8vcmFwdG9yL1plbmh1YkVwaWMvMTQzNzMy)

timeline: 2023/Q4 - 2024/Q1

right now catching this as an issue for #177

- [x] logging solution in #405
- [ ] log sources this issue

### PR DESCRIPTION
**Pull request by bnshr**
This PR configures the test cluster to use the LOKI installed in the obs cluster

fixes https://github.com/nerc-project/operations/issues/324
Referring to https://github.com/nerc-project/operations/issues/324

### CODE REVIEW

**Review by schwesig — COMMENTED** [review]
looking good,
but want to wait on a second review (from @computate) in the meeting today

**Review by computate — APPROVED** [review]

**Review by schwesig — APPROVED** [review]

**Pull request merged**
